### PR TITLE
Adds 3 functions to list tags from root anchor and any anchor type and addresses of anchor-types from root

### PR DESCRIPTION
**Pull request by philipbeadle**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -45,8 +45,8 @@ pub fn get_anchor(address: Address) -> ZomeApiResult<Anchor> {
     hdk::utils::get_as_type(address)
 }
 
-/// Gives a list of all anchors.
-pub fn get_anchors() -> ZomeApiResult<Vec<Address>> {
+/// Gives a list of all anchor types.
+pub fn get_anchor_types() -> ZomeApiResult<Vec<Address>> {
```
Comment (freesig) [inline src/lib.rs:49]: This is returning the address of the `Anchor` though?

File: src/lib.rs [hunk 2]
```diff
@@ -57,6 +57,37 @@ pub fn get_anchors() -> ZomeApiResult<Vec<Address>> {
...
+pub fn list_anchor_type_addresses() -> ZomeApiResult<Vec<Address>> {
+    let root_anchor_address = root_anchor()?;
+    Ok(hdk::get_links(
+        &root_anchor_address,
+        LinkMatch::Exactly(ANCHOR_LINK_TYPE),
+        LinkMatch::Any,
+    )?
+    .addresses()
+    .to_owned())
+}
```
Comment (freesig) [inline src/lib.rs:70]: This is exactly the same as above?

File: src/lib.rs [hunk 3]
```diff
@@ -57,6 +57,37 @@ pub fn get_anchors() -> ZomeApiResult<Vec<Address>> {
...
+pub fn list_anchor_types() -> ZomeApiResult<Vec<String>> {
+    let root_anchor_address = root_anchor()?;
+    Ok(hdk::get_links(&root_anchor_address, LinkMatch::Exactly(ANCHOR_LINK_TYPE), LinkMatch::Any)?.links()
+    .iter()
+    .map(|link| link.tag.clone())
+    .collect())
+}
+
+/// Gives a list of all anchor_type's anchors.
+pub fn list_anchor_type_texts(anchor_type: String) -> ZomeApiResult<Vec<String>> {
```
Comment (freesig) [inline src/lib.rs:82]:
```suggestion
/// Gives a list of all anchor's anchor_texts.
pub fn list_anchor_texts(anchor_type: String) -> ZomeApiResult<Vec<String>> {
```

File: src/lib.rs (hunk 1 above)
Comment (philipbeadle) [reply src/lib.rs:49]: it returns the list of addresses of the anchor types linked to the root anchor.

File: src/lib.rs (hunk 2 above)
Comment (philipbeadle) [reply src/lib.rs:70]: ah that should be deleted

File: src/lib.rs (hunk 1 above)
Comment (freesig) [reply src/lib.rs:49]: ah I see yep

**Pull request merged**